packages/salesforcedx-vscode-core/test/jest/settings/sfdxCoreSettings.test.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2023, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { ENABLE_SOURCE_TRACKING_FOR_DEPLOY_RETRIEVE } from '../../../src/constants';
+import { SfdxCoreSettings } from '../../../src/settings/sfdxCoreSettings';
+
+describe('sfdxCoreSettings', () => {
+  const sfdxCoreSettingsInstance = SfdxCoreSettings.getInstance();
+  let getConfigValueSpy: jest.SpyInstance;
+  getConfigValueSpy = jest.spyOn((SfdxCoreSettings as any).prototype, 'getConfigValue');
```

Comment:
You should put this in a beforeEach. Otherwise as tests get added this file the spy will be in a weird state between tests.